src/runtime/propensity.js

```diff
@@ -29,15 +29,17 @@ export class Propensity {
    * @param {!Window} win
    * @param {!../model/page-config.PageConfig} pageConfig
    * @param {!../api/client-event-manager-api.ClientEventManagerApi} eventManager
+   * @param {!./fetcher.Fetcher} fetcher
    */
-  constructor(win, pageConfig, eventManager) {
+  constructor(win, pageConfig, eventManager, fetcher) {
```

Comment:
I would have expected changes to Propensity-test.js where the Propensity object is created.

Reply:
Added it, it never actually calls the fetcher in propensity-test because it's using a fake propensity-server. Fetcher is tested in propensity-server-test